maro/simulator/scenarios/vm_scheduling/data_center.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+
+
+@node("data_centers")
+class DataCenter(NodeBase):
+    """Cluster node definition in frame."""
+    id = NodeAttribute("i2")
+    region_id = NodeAttribute("i2")
+    zone_id = NodeAttribute("i2")
+
+    total_machine_num = NodeAttribute("i")
```

Comment:
no empty_machine_num for this class?

Reply:
Referring to the Protean structure, the decision is made at the cluster level, so I think the upper level should not keep the empty machine amount?

Reply:
In the implementation of BE, we don't stipulate that we can only make decisions at the zone level. So it can enable more decision levels.

Also for the visualization, A global view of machine utilization may be useful?